src/utils/distribcell.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python2
```

Comment:
This file is really out of date.  You could try to update it to the Python API and make it more usable, but I would recommend just deleting it.